aws/continuous_integration/ci_aws_uploader.rb

```diff
@@ -0,0 +1,45 @@
+class CIAWSUploader
```

Comment:
I think we prefer treating acronyms as words for camelcasing, so `CiAwsUploader` here